### PR TITLE
Document OPA UI

### LINKED ISSUE #600: Document how to enable & use OPA in the UI
Document how to enable & use OPA in the UI, probably as a tutorial

### PR DESCRIPTION
**Pull request by kgroschoff**
Fixes #600

### CODE REVIEW

File: content/kubermatic/master/tutorials/opa/_index.en.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        msg := sprintf("you must provide labels: %v", [missing])
+      }
+```
+
+Just click on `Add` to create the constraint template. In this table you can also edit or delete it again if needed.
+
+![Constraint Templates](admin-ct-overview.png)
+
+
+### Activating OPA during Cluster Creation
```
Comment (lsviben) [inline content/kubermatic/master/tutorials/opa/_index.en.md:69]: i would put the OPA activating options together, and OPA operating stuff together. Now there is not a clear connection between the Constraint Templates and the Constraints.

File: content/kubermatic/master/tutorials/opa/_index.en.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+- `OPA Integration` in the top area is indicating if OPA is enabled or not.
+- `OPA Gatekeeper Controller` and `OPA Gatekeeper Audit` provide information about the status of those controllers.
+- `OPA Constraints` and `OPA Gatekeeper Config` are added to the tab menu on the bottom. More details in the following sections.
+
+![Cluster Details View](cluster-overview.png)
+
+
+### Constraints
+
+Constraints basically are a collection of rules that describe the expected state of your service. They need to be associated with a constraint template.
```
Comment (lsviben) [inline content/kubermatic/master/tutorials/opa/_index.en.md:100]: Constraints are the filler for rules that are defined by the constraint templates. Constraints provide the paramenters which are used in the CT rule. We could say that it specifies the rule.

File: content/kubermatic/master/tutorials/opa/_index.en.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+![Cluster Details View](add-constraint.png)
+
+Following example will make sure that the gatekeeper label is defined on all namespaces, if you are using the `K8sRequiredLabels` constraint template from above:
+```
+match:
+  kinds:
+    - apiGroups: [""]
+      kinds: ["Namespace"]
+parameters:
+  labels: ["gatekeeper"]
```
Comment (lsviben) [inline content/kubermatic/master/tutorials/opa/_index.en.md:115]: This wont work, it should be like:
```suggestion
parameters:
  rawJSON: '{"labels":["gatekeeper"]}'
```
```suggestion
parameters:
  labels: ["gatekeeper"]
```

**Review by lsviben — CHANGES_REQUESTED** [review]

**Comment by kgroschoff** [comment]
Thanks @lsviben, I tried to restructure it a bit. You're absolutely right, that the order was not really easy to understand for a stranger. Also I liked your sentence that explains a constraint; I hope it's okay that I copied it?
PTAL :)

**Pull request merged**